src/psycop/common/model_evaluation/confusion_matrix/confusion_matrix.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import dataclass
+
+import pandas as pd
+
+
+@dataclass
+class ConfusionMatrix:
+    true_positives: int
+    true_negatives: int
+    false_positives: int
+    false_negatives: int
+
+    @property
+    def ppv(self) -> float:
+        return self.true_positives / (self.true_positives + self.false_positives)
+
+    @property
+    def npv(self) -> float:
+        return self.true_negatives / (self.true_negatives + self.false_negatives)
+
+    @property
+    def sensitivity(self) -> float:
+        return self.true_positives / (self.true_positives + self.false_negatives)
+
+    @property
+    def specificity(self) -> float:
+        return self.true_negatives / (self.true_negatives + self.false_positives)
+
+
+def get_confusion_matrix_cells_from_long_df(
+    long_df: pd.DataFrame,
+) -> ConfusionMatrix:
+    """Get confusion matrix cells from a long dataframe."""
```

Comment:
Please add a comment on the required columns/format - long_df is a bit too unspecific. It also seems that the df should actually be wide with columns "true" and "pred"?

Reply:
EDIT2, or not? - is `create_long_confusion_matrix_df` supposed to be called by the user? In that case I would move it away from `test_confusion_matrix.py` and rather have it in this file.